utils/src/mesh.rs

```diff
@@ -179,3 +193,16 @@ impl KaboodleMesh for ServalMesh {
             .collect()
     }
 }
+
+pub fn mesh_interface_and_port() -> (Option<if_addrs::Interface>, u16) {
+    let mesh_port: u16 = match std::env::var("MESH_PORT") {
```

Comment:
I just touched this logic earlier this morning and am happy to drop my commit so we don’t conflict, but in case you want it, I thought this was a bit easier to read than our original port parsing code:

````rs
let mesh_port: u16 = std::env::var("MESH_PORT")
    .ok()
    .map(|port_str| port_str.parse().expect("Invalid value given for MESH_PORT"))
    .unwrap_or(8181);
````